README.md

```diff
@@ -93,4 +118,57 @@ List of Kubernetes nodes:
 - docker-for-desktop - map[beta.kubernetes.io/arch:amd64 beta.kubernetes.io/os:linux kubernetes.io/hostname:docker-for-desktop node-role.kubernetes.io/master:]
 ```
 
+### `kurun` is like `kubectl port-forward` to Kubernetes (and not out!)
```

Comment:
this sentence is not clear to me

Reply:
Would be

##### `kurun` is like `kubectl port-forward` into Kubernetes (and not out from!)

more clear?

Reply:
yes